docs/steamdeck-and-linux/installing-on-steamdeck-and-linux.md

```diff
@@ -31,6 +31,50 @@ This guide assumes you're *up to date with NorthstarProton*, as the method used
 If you're using an older version of NorthstarProton, replace the information about launch options with a text file called `run_northstar.txt` in your Titanfall2 directory. This text file should have a single character `1` inside.
 {% endhint %}
 
+### Ubuntu / Debian
+This workaround employs the use of [Distrobox](https://github.com/89luca89/distrobox) to create a Fedora instance and install Steam to it. Your Fedora instance created through this method will share the same Steam files that the native package uses, so you won't have to re-download Titanfall 2.
+
+1. Open a terminal and install Distrobox with `sudo apt install distrobox`
+2. Run `distrobox-create --image fedora:latest --name Fedora`
+3. Enter your newly created instance with `distrobox-enter Fedora`
+4. Enable `rpm-fusion` by running `sudo dnf install https://download1.rpmfusion.org/free/fedora/rpmfusion-free-release-$(rpm -E %fedora).noarch.rpm`
+5. Install Steam by running `sudo dnf install steam`
+6. Export Steam out of your instance by running `distrobox-export steam`, this creates a `.desktop` file outside of the instance that allows Steam to be ran in the container from a shortcut, allowing you to run it without needing to enter your Fedora instance every time.
+7. Search for `Steam (Runtime) (on Fedora)` in your Applications, and optionally pin it to your taskbar.
+8. If you haven't already, follow the steps above to install NorthstarProton.
+
+We recommend keeping your Fedora instance up to date, which can be done manually by running `sudo dnf update` inside of your Fedora instance.
```

Comment:
Maybe a subheader or something like this to indicate that this is not immediately necessary to get Northstar up and running but should considered for the long run?

```suggestion
**Updating distrobox container: (optional)**

We recommend keeping your Fedora instance up to date, which can be done manually by running `sudo dnf update` inside of your Fedora instance.
```